WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartRepository.kt

```diff
@@ -129,6 +129,18 @@ class QuickStartRepository
         }
     }
 
+    fun skipQuickStart() {
+        selectedSiteRepository.getSelectedSite()?.let { site ->
+            val siteId = site.id.toLong()
+            for (quickStartTask in QuickStartTask.values()) {
+                quickStartStore.setDoneTask(siteId, quickStartTask, true)
+            }
```

Comment:
Suggestion (💡): This `for/in` can be replaced with a one-liner `forEach`, see below:
```
QuickStartTask.values().forEach { quickStartStore.setDoneTask(siteLocalId, it, true) }
```

PS: The same applies to that exact test that is utilizing `for/in`.

Reply:
In 4f1b657